ci/src/test/java/group10/CloneTest.java

```diff
@@ -0,0 +1,57 @@
+package group10;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Test cloning a repository
+ */
+public class CloneTest {
+
+  CIServer ci;
+  File cloneDirectoryPath;
+
+  @Before
+  public void setUp() {
+    ci = new CIServer();
+    cloneDirectoryPath = new File("../../gitclones");
+  }
```

Comment:
This cloneDirectoryPath places the test repository outside of the git repository: `DD2480_A2_CI_Group10/ci/../../gitclones`.  Tests are executed from `ci/` so perhaps change the path or use JUnit TemporaryFolder instead. JUnit creates this folder and cleans it up after running the tests.

Reply:
I changed the path